app/services/statement_creator.rb

```diff
@@ -17,8 +17,8 @@ def create
       set_order_details_to_statement
       setup_statement_from_details
       raise ActiveRecord::Rollback if errors.any?
+      LogEvent.log(@statement, :create, session_user)
```

Comment:
It's possible for this statement creator to create multiple statements. It creates one for each account. I think we need to either put this in the loop where the statement gets created or we might be able to loop over `@account_statements.values` (that variable appears to be a mapping of account => statement.